src/components/skills-quiz/CurrentJobDropdown.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useContext, useMemo } from 'react';
+import { SearchContext } from '@edx/frontend-enterprise-catalog-search';
+import FacetListRefinement from '@edx/frontend-enterprise-catalog-search/FacetListRefinement';
+
+import { CURRENT_JOB_FACET } from './constants';
+
+const CurrentJobDropdown = () => {
+  const { refinements } = useContext(SearchContext);
+  const {
+    title, attribute, typeaheadOptions, facetValueType, customAttribute,
+  } = CURRENT_JOB_FACET;
+  const currentJobDropdown = useMemo(
+    () => {
+      const filtersFromRefinements = () => (
+        <FacetListRefinement
+          key={attribute}
+          title={title}
+          attribute={attribute}
+          limit={300} // this is replicating the B2C search experience
+          refinements={refinements}
+          facetValueType={facetValueType}
+          typeaheadOptions={typeaheadOptions}
+          searchable={!!typeaheadOptions}
+          doRefinement={false}
+          customAttribute={customAttribute}
+        />
+      );
+      return (
+        <>
+          {filtersFromRefinements()}
+        </>
+      );
+    },
+    [JSON.stringify(refinements)],
```

Comment:
can we use `refinements` directly here?

Reply:
I think no, the format is changed now and they are passed to SearchContext in this format to set in URL and Context.